### PR TITLE
[new release] bimage, bimage-unix, bimage-io and bimage-display (0.5.0)

### PR DESCRIPTION
**Pull request by zshipko**
A simple, efficient image-processing library

- Project page: <a href="https://github.com/zshipko/ocaml-bimage">https://github.com/zshipko/ocaml-bimage</a>
- Documentation: <a href="https://zshipko.github.io/ocaml-bimage/doc">https://zshipko.github.io/ocaml-bimage/doc</a>

##### CHANGES:

- Removed `Bimage_unix.Thread.Filter`
- Renamed `Bimage_unix.{Data_unix,Image_unix}.create_mmap` to `mmap`
- Removed `Filter.Make`, `FILTER` and `bimage-lwt`

### CODE REVIEW

**Comment by mseri** [comment]
I think you should use `available: os = "linux"` instead of constraining ocaml

File: packages/bimage-display/bimage-display.0.5.0/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+homepage: "https://github.com/zshipko/ocaml-bimage"
+doc: "https://zshipko.github.io/ocaml-bimage/doc"
+license: "ISC"
+dev-repo: "git+https://github.com/zshipko/ocaml-bimage.git"
+bug-reports: "https://github.com/zshipko/ocaml-bimage/issues"
+tags: ["image processing"]
+
+depends:
+[
+    "ocaml" {>= "4.08.0" & os = "linux"}
```
Comment (mseri) [inline packages/bimage-display/bimage-display.0.5.0/opam:13]:
```suggestion
    "ocaml" {>= "4.08.0"}
```

File: packages/bimage-display/bimage-display.0.5.0/opam [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+]
+
+synopsis: """
+Window system for Bimage
+"""
+
+description: """
+Allows for Bimage Images to be displayed using OpenGL
+"""
+url {
```
Comment (mseri) [inline packages/bimage-display/bimage-display.0.5.0/opam:31]:
```suggestion

available: os = "linux"

url {
```

File: packages/bimage-io/bimage-io.0.5.0/opam [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+homepage: "https://github.com/zshipko/ocaml-bimage"
+doc: "https://zshipko.github.io/ocaml-bimage/doc"
+license: "ISC"
+dev-repo: "git+https://github.com/zshipko/ocaml-bimage.git"
+bug-reports: "https://github.com/zshipko/ocaml-bimage/issues"
+tags: ["image processing"]
+
+depends:
+[
+    "ocaml" {>= "4.08.0" & os = "linux"}
```
Comment (mseri) [inline packages/bimage-io/bimage-io.0.5.0/opam:13]:
```suggestion
    "ocaml" {>= "4.08.0"}
```

File: packages/bimage-io/bimage-io.0.5.0/opam [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+]
+
+synopsis: """
+Input/output for Bimage using OpenImageIO
+"""
+
+description: """
+Input/output for Bimage using OpenImageIO
+"""
+url {
```
Comment (mseri) [inline packages/bimage-io/bimage-io.0.5.0/opam:30]:
```suggestion

available: os = "linux"

url {
```

**Review by mseri — CHANGES_REQUESTED** [review]

**Comment by zshipko** [comment]
Thanks! I knew there was something like that but couldn't remember the syntax

**Comment by mseri** [comment]
Thanks

**Pull request merged**